bench/styles/index.html

```diff
@@ -8,12 +8,15 @@
 
     <link href='https://www.mapbox.com/base/latest/base.css' rel='stylesheet' />
     <link rel='stylesheet' href='/dist/mapbox-gl.css'/>
-    <script href='https://unpkg.com/url-search-params@0.10.0/build/url-search-params.js'></script>
 </head>
 
 <body>
     <div id="benchmarks"></div>
     <script src="/bench/styles/benchmarks_generated.js"></script>
+    <script src="/bench/benchmarks_view_generated.js"></script>
+    <script>
+        window.Benchmarks.run(benchmarks);
```

Comment:
Is the `benchmarks` variable provided by benchmarks_view_generated.js?

Reply:
Trying to think through any consequences of calling `Benchmark.run` for my submodule use case of the benchmark suite. I think its probably fine but I'm just curious the reasoning here.

Reply:
@tristen it's provided by `index.html` and `benchmarks.js` of each respective benchmark. Styles and versions benchmarks need different treatment here — versions benchmarks group by name in a global variable because versions are ran as separate builds (and we need to keep compat with previous benchmark builds), and styles benchmarks build the benchmarks array in the build itself (because we don't upload them, and they need additional metadata like location).